cves/CVE-2015-6826.yml

```diff
@@ -342,7 +381,28 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer:
+  answer: |
+    The vulnerability was likely caused by a lack of understanding of the
+    system coupled with a lack of robust regression testing, which would have
+    potentially caught the vulnerability the moment it was introduced.
+
+    The function in question was stripped of a lot of its functionality in the
+    VCC, but nothing equivalent to the stripped functionality was reintroduced.
+    A greater understanding of the system as a whole would likely have revealed
+    that there was a possible thread of execution that did not initialize or
+    free the struct fields anywhere else.
+
+    It is also interesting to note that the fix did not suggest a sufficient
+    understanding of the code, either. The VCC itself included a helper
+    function that appears to do exactly what the fix accomplished, but the
+    helper function was not used in either the VCC or the fix within the
+    function in question.
```

Comment:
Was the helper function used at all elsewhere in the code? Did it still exist at the time of the fix commits?

Reply:
I'll check to be sure, but I do believe the helper function was used once somewhere else in the file. I'll make an update if so.

Reply:
Turns out it was used a few times. Update added